docs/connected_orgs.rst

```diff
@@ -1,14 +1,133 @@
+
+    Example: A single persistent org connected to CumulusCI with the name ``devhub``.
+
+.. code-block:: yaml
+
+    $ cci org list
+
+    ┌Scratch Orgs──────────────┬─────────┬──────┬─────────┬──────────────┬────────┐
+    │ Name                     │ Default │ Days │ Expired │ Config       │ Domain │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ dev                      │         │ 7    │ X       | dev          │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ feature                  │         │ 1    │ X       | feature      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ prerelease               │         │ 1    │ X       | prerelease   │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ qa                       │         │ 7    │ X       | qa           │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ release                  │         │ 1    │ X       | release      │        │
+    └──────────────────────────┴─────────┴──────┴─────────┴──────────────┴────────┘
+
+    ┌Connected Orgs────┬──────────────────────────────┬────────────┐
+    │ Name   │ Default │ Username                     │ Expires    │
+    ├────────┼─────────┼──────────────────────────────┼────────────┤
+    │ devhub │         │ peter.gibbons@initech.devhub │ Persistent │
+    └────────┴─────────┴──────────────────────────────┴────────────┘
+
+Make sure that CumulusCI can login to the connected org.
+
+.. code-block:: console
+
+    $ cci org browser <org_name>
+
+
+
+Global Orgs
+-----------
+
+By default, ``cci org connect`` stores the OAuth credentials for connected orgs in a *project specific* keychain. These credentials ensure that an org connected in Project A's directory isn't accessible when you're working in Project B's directory.
+
+Connect an org and make it available to *all* CumulusCI projects on your computer by passing the ``--global-org`` flag.
+
+.. code-block:: console
+
+    $ cci org connect <org_name> --global-org
+
+
+
+Use a Custom Connected App
+----------------------------
+
+CumulusCI uses a preconfigured Connected App to authenticate to Salesforce orgs that use OAuth2. In most cases this works fine. To control the Connected App for specific security or compliance requirements, create your own Connected App and configure CumulusCI to use it when connecting to orgs.
+
+To create a custom Connected App, run the ``connected_app`` task to create the Connected App, and then manually edit its configuration to suit your requirements.
+
+.. important :: Make sure to create the Connected App in a persistent org other than a sandbox!
+
+    Example: Create a Connected App in the DevHub org connected to ``SFDX`` with the label ``cumulusci`` and set it as the ``connected_app`` service in CumulusCI.
```

Comment:
I believe this needs the breaking symbol `..` above the example. Currently the example is rendering as part of the 'important' admonition above.